bootstrap.lic

```diff
@@ -3,9 +3,29 @@
   Documentation: https://elanthipedia.play.net/Lich_script_development#bootstrap
 =end
 
-class_defs = { 'equipmanager' => :EquipmentManager, 'common' => :DRC, 'common-travel' => :DRCT, 'common-crafting' => :DRCC, 'common-summoning' => :DRCS, 'common-money' => :DRCM, 'common-arcana' => :DRCA, 'common-items' => :DRCI, 'common-healing' => :DRCH, 'common-validation' => :CharacterValidator, 'common-moonmage' => :DRCMM, 'common-theurgy' => :DRCTH }
+$BOOTSTRAP_VERSION = '1.0.1'
 
-arg_definitions = [[{ name: 'wipe_constants', regex: /wipe_constants/i, optional: true }]]
+class_defs = {
+  'drinfomon' => :DRINFOMON,
```

Comment:
This is new; the rest of `class_defs` has been formatted for readability; the line was super long

Reply:
Thanks for these notes. Makes it clear what's actually new.